ingestion_tools/scripts/data_validation/standardized/fixtures/path.py

```diff
@@ -84,14 +84,24 @@ def frames_files(frames_dir: str, filesystem: FileSystemApi) -> List[str]:
     """[Dataset]/[ExperimentRun]/Frames/*"""
     files = filesystem.glob(f"{frames_dir}/*")
     # Exclude mdoc files, add s3 prefix
-    refined_files = ["s3://" + file for file in files if ".mdoc" not in file]
+    refined_files = ["s3://" + file for file in files if ".mdoc" not in file and ".json" not in file]
     # mdoc files are in the folder, but just no frames
     if len(refined_files) == 0 and len(files) != 0:
         pytest.skip(f"No frame files in frames directory: {frames_dir}")
 
     return refined_files
 
 
+@pytest.fixture(scope="session")
+def frames_meta_file(frames_dir: str, filesystem: FileSystemApi) -> str:
+    """[Dataset]/[ExperimentRun]/Frames/frames_metadata.json"""
+    dst = f"{frames_dir}/frames_metadata.json"
+    if filesystem.exists(dst):
+        return dst
+    else:
+        pytest.fail(f"Frames metadata file not found: {dst}")
```

Comment:
Could we update the message such that, it informs that  Frames directory exists but the metadata.json doesn't?